.github/workflows/build-matrix.json

```diff
@@ -0,0 +1,49 @@
+[
+    {
```

Comment:
It is only node's matrix, right? please add it to the name, e.g. node-build-matrix

Reply:
maybe we're better of having some folder for all future matrices

Reply:
just saw it is being used also for pypi, see:
https://github.com/aws/glide-for-redis/pull/1467/files#r1618848689